src/sds_data_model/_vector.py

```diff
@@ -172,3 +174,23 @@ def _from_delayed_to_data_array(
         name=name,
         attrs=_metadata,
     )
+
+def _check_projection(
+    vector_data: str, **data_kwargs
```

Comment:
I don't think these parameter names accurately describe the arguments the function takes.

You're expecting a string path not some vector data and you don't need to clarify these key word arguments with a prefix because you're only expecting one set.

```suggestion
    path: str, **kwargs
```